internal/metastructure/target_update/target_update.go

```diff
@@ -0,0 +1,79 @@
+// © 2025 Platform Engineering Labs Inc.
+//
+// SPDX-License-Identifier: FSL-1.1-ALv2
+
+package target_update
+
+import (
+	"time"
+
+	"github.com/platform-engineering-labs/formae/internal/api/model"
+	"github.com/platform-engineering-labs/formae/internal/metastructure/types"
+	"github.com/platform-engineering-labs/formae/internal/metastructure/util"
+	pkgmodel "github.com/platform-engineering-labs/formae/pkg/model"
+)
+
+type (
+	TargetOperation   = types.OperationType
```

Comment:
nit: unrelated to this PR but I noticed the comment on this `OperationType` constant is outdated, we should probably update it that this now pertains to resource updates as well as target updates